markdown/guide/addons.md

```diff
@@ -23,6 +23,16 @@ ember-engines will automatically try to deduplicate addons used by your host app
 
 If addons are not included in the host app, they will be included in the engine's vendor bundle. 
 
+Isolation and liability is generally hard to solve, all possible cases have down sides here:
```

Comment:
"and liability"?

Reply:
yes, ember engines helps in separating concepts and responsibilities right?

I put liability as synonym of responsibility

Reply:
fixed!